harness/determined/cli/agent.py

```diff
@@ -108,7 +117,14 @@ def get_task_name(containers: Dict[str, Any], slot: bindings.v1Slot) -> str:
     slots = [
         collections.OrderedDict(
             [
-                ("agent_id", local_id(agent.id)),
```

Comment:
blocking: reaching into this container feels weird to me since we will have a different value on it the slot has a container or not for agent?

Was there a reason or issue with keeping the agent_id the same as it was before this change?

Reply:
No issue necessarily, but I changed it for the sake of keeping sensitive information obfuscated.  On line 70 of `agent.py`, we call `get_GetAgents`, where agent information is obfuscated. In the first PR for this ticket [here](https://github.com/determined-ai/determined/commit/62084e260696b99536a15e3ef8445b76ccff3807), I did change `ObfuscateAgent` to obscure agent Id after talking to @corban-beaird about optimal security, so I mostly added the slot container check here because this is for `det slot list` and not `det agent list`. But I can definitely remove it if it's unnecessary / redundant.

Reply:
I'm gonna push back a little against changing agentID to also be obfuscated in this PR

I think this ticket is addressing a specific customer ask about our confusing display of RBAC denied tasks in ```det slot``` not about agentIDs being exposed.

We should probably get product input too if we plan on trying to obfuscate change agentIDs separately since it is a usability verse security question.

Reply:
Gotchya, agreed that it does not directly address the issue described in the ticket, so I changed this back!